SignalServiceKit/src/Storage/TSStorageManager.h

```diff
@@ -18,22 +12,15 @@ NS_ASSUME_NONNULL_BEGIN
 
 + (instancetype)sharedManager;
 
-/**
- * The safeBlockingMigrationsBlock block will
- * run any outstanding version migrations that are a) blocking and b) safe
- * to be run before the environment and storage is completely configured.
- *
- * Specifically, these migration should not depend on or affect the data
- * of any database view.
- */
-- (void)setupDatabaseWithSafeBlockingMigrations:(void (^_Nonnull)(void))safeBlockingMigrationsBlock;
-
-// TODO: Deprecate.
+- (YapDatabaseConnection *)dbReadConnection;
+- (YapDatabaseConnection *)dbReadWriteConnection;
 + (YapDatabaseConnection *)dbReadConnection;
 + (YapDatabaseConnection *)dbReadWriteConnection;
 
 + (void)migrateToSharedData;
 
++ (NSString *)databaseFilePath;
```

Comment:
No biggie, but I don't see why this needs to be public. Can we get away with it only in +Subclass.h ?

Reply:
Let's revisit in the next PR where I continue to rework this class.